src/web/src/components/Posts/PostAvatar.tsx

```diff
@@ -48,6 +48,8 @@ const PostAvatar = ({ name, img, url }: AvatarProps) => {
 
   if (name.length > 0) {
     const initials = name
+      .replace(/[^a-zA-Z ]*/, '')
```

Comment:
Did you test this?  It still does the same thing with `David Humphrey (YouTube)`.  You need to add `/g` to this regex to deal with all matches globally.

Reply:
@humphd can you explain on how to test it ? cause I trying to test the your Youtube.

Reply:
my fault, im having a little trouble getting the video feeds showing up. will update it with test screenshots